frame/system/src/lib.rs

```diff
@@ -628,11 +649,15 @@ impl<T: Trait> Module<T> {
 	}
 
 	/// Start the execution of a particular block.
+	///
+	/// Setting `is_for_inspection` flag will not cause the events
```

Comment:
I think the comment can be removed? Or at least be fixed.

Reply:
Good catch, thanks!